test/functions_test.py

```diff
@@ -64,14 +68,19 @@
 <BLANKLINE>
 Message: V60 method
 >>> file.close()
->>> save_result(result_params, "save_test1.json", "json")
+>>> save_details = save_result(result_params, "save_test1.json", "json")
+>>> save_details["status"]
+True
+>>> save_details["message"] == "[Info] File saved successfully!"
+True
 >>> file = open("save_test1.json", "r")
 >>> save_test1_object = json.load(file)
->>> save_test1_object == {'mycoffee_version': MY_COFFEE_VERSION, 'water_unit': 'g', 'temperature': 93, 'method': 'v60', 'water': 500, 'coffee_unit': 'g', 'coffee_ratio': 3, 'cups': 2, 'digits': 3,'coffee': 60, 'message': 'V60 method', 'temperature_unit': 'C', 'grind': 500, 'grind_unit': 'um', 'warnings': [], 'water_ratio': 50}
+>>> save_test1_object == {'mycoffee_version': MY_COFFEE_VERSION, 'water_unit': 'g', 'temperature': 93, 'method': 'v60', 'water': 500, 'coffee_unit': 'g', 'coffee_ratio': 3, 'cups': 2, 'digits': 3,'coffee': 60, 'message': 'V60 method', 'temperature_unit': 'C', 'grind': 500, 'grind_unit': 'um', 'warnings': [], 'water_ratio': 50, 'grind_type': get_grind_type(500)}
```

Comment:
`'grind_type': get_grind_type(500)` this looks odd a bit. Should it be saved like a raw function call?

Reply:
No.
```pycon
>>> from mycoffee.functions import *
>>> get_grind_type(500)
'Medium-Fine'
```
I use this method in tests to ensure more reproducibility.